### DISCUSSION TITLE
Is it possible to enable cookie consent for books?

### DISCUSSION THREAD

**Discussion by lballabio** [quarto-dev/quarto-cli#11559] [Q&A] [answered]
### Description

Hello—am I right in understanding that `cookie-consent` only works for a Quarto website and not for the HTML output of a Quarto book?  I tried adding `cookie-consent` to the `book` section of my YML but it raises an error saying it's an invalid property there.  On the other hand, the YML seems to accept the property if I add it under `format: html:` but this doesn't seem to have any effect on the output.  Thanks!

**Comment by mcanouil** [comment]
Could you share the YAML you tried and where you put it?

**Comment by lballabio** [comment, marked as answer]
Never mind—it wasn't working in Quarto 1.5.57 but it works after upgrading to 1.6.39.  Thanks anyway.
